validated-method.js

```diff
@@ -96,12 +97,10 @@ perhaps you meant to throw an error?`);
 
 // Mixins get a chance to transform the arguments before they are passed to the actual Method
 function applyMixins(args, mixins) {
-  // You can pass nested arrays so that people can ship mixin packs
-  const flatMixins = _.flatten(mixins);
```

Comment:
I just removed this, since nested arrays have actually never been allowed because of the check in the constructor: https://github.com/meteor/validated-method/blob/b94d376ec438e6d3589b86449ef97901e2fd0ee7/validated-method.js#L7 .

Maybe we could add it later once 'flat' is supported in Meteor (or is it already)? but since nobody has used it up to now...